src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/HardcodedEncryptionKeySources.cs

```diff
@@ -28,16 +29,30 @@ static HardcodedEncryptionKeySources()
                 WellKnownTypeNames.SystemConvert,
                 isInterface: false,
                 taintedProperties: null,
-                taintedMethodsNeedPointsToAnalysis: null,
-                taintedMethodsNeedsValueContentAnalysis: new (string, IsInvocationTaintedWithValueContentAnalysis)[]{
+                taintedMethodsNeedsPointsToAnalysis: null,
+                taintedMethodsNeedsValueContentAnalysis: new (string, IsInvocationTaintedWithValueContentAnalysis[])[]{
                     ("FromBase64String",
-                    (IEnumerable<PointsToAbstractValue> argumentPonitsTos, IEnumerable<ValueContentAbstractValue> argumentValueContents) => argumentValueContents.All(o => o.IsLiteralState)),
+                    new IsInvocationTaintedWithValueContentAnalysis[]{
+                        (IEnumerable<IArgumentOperation> arguments, IEnumerable<PointsToAbstractValue> argumentPonitsTos, IEnumerable<ValueContentAbstractValue> argumentValueContents) => argumentValueContents.All(o => o.IsLiteralState),
+                    }),
+                });
+            builder.AddSourceInfoSpecifyingTaintedTargets(
+                WellKnownTypeNames.SystemTextEncoding,
+                isInterface: false,
+                taintedProperties: null,
+                taintedMethodsNeedsPointsToAnalysis: null,
+                taintedMethodsNeedsValueContentAnalysis: new (string, (IsInvocationTaintedWithValueContentAnalysis, string[])[])[]{
+                    ("GetBytes",
+                    new (IsInvocationTaintedWithValueContentAnalysis, string[])[]{
+                        ((IEnumerable<IArgumentOperation> arguments, IEnumerable<PointsToAbstractValue> argumentPonitsTos, IEnumerable<ValueContentAbstractValue> argumentValueContents) => arguments.Count() == 5 && argumentValueContents.FirstOrDefault().IsLiteralState, new[] { "bytes", }),
+                        ((IEnumerable<IArgumentOperation> arguments, IEnumerable<PointsToAbstractValue> argumentPonitsTos, IEnumerable<ValueContentAbstractValue> argumentValueContents) => arguments.Count() == 1 && argumentValueContents.FirstOrDefault().IsLiteralState, new[] { "RETURN", }),
```

Comment:
>RETURN [](start = 278, length = 6)

Instead of "RETURN" everywhere, let's have a constant string defined, and also make sure it's an illegal identifier name in .NET (like ".RETURN").